generator-prisma-client/bun-with-prisma-driver-adapters/.env.example

```
@@ -0,0 +1,2 @@
+DATABASE_URL=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)